src/templates/so-clean/components/pagination.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+	<nav class="pagination" role="navigation">
+		<a class="newer-posts" @click="callback(page - 1)" v-if="page>1"><span aria-hidden="true">&larr;</span> 新点儿的</a>
```

Comment:
Better lint code in vue expressions too, by yourself. Add spaces before and after operators.